nautobot/dcim/models/devices.py

```diff
@@ -998,3 +1013,69 @@ def to_csv(self):
             self.domain,
             self.master.name if self.master else None,
         )
+
+
+@extras_features(
+    "custom_fields",
+    "custom_links",
+    "custom_validators",
+    "dynamic_groups",
+    "export_templates",
+    "graphql",
+    "relationships",
+    "statuses",
+    "webhooks",
+)
+class RedundancyGroup(PrimaryModel, ConfigContextModel, StatusModel):
+    """
+    A RedundancyGroup represents a logical grouping of physical hardware for the purposes of indicating redundancy pairs of high-availability.
+    """
+
+    name = models.CharField(max_length=100, unique=True)
```

Comment:
Should we also include a `slug` and `description` for RDs? What's the thought process here between `description` and/or `comments` or both?

Reply:
`slug` is currently in contention? Name is unique so should we use `natural_key` instead?